test/home.test.js

```diff
@@ -13,33 +13,56 @@ function waitFor(timeout) {
 
 const nextBin = join(__dirname, '..', 'node_modules', '.bin', 'next')
 const cwd = join(__dirname, '..')
+const env = {
+  ...process.env,
+  NODE_ENV: 'production',
+  SESSION_KEY: '8c415823f70769700a1440dc3b871ecb7666e481332a874a22ad2595f954fef13c37ffd1cd9eb6c49d99258fe9e3f8ad841350a5523c3f86c726fe2a34bb1020',
+  ACCESS_CODE: '030245c3d496d0811636a7b3dfeebc5b33c9804eb1053feb3460744ae7b8f5435d629bd0ce552ea1846a3f95e39f1b6fdea5293809c71f3779313c3ee3f60d75'
+}
 
 async function startApp() {
-  const instance = spawn(
-    '/usr/bin/env',
-    ['NODE_ENV=production', 'node', 'server.js'],
-    { stdio: 'inherit', cwd }
+  const server = spawn(
+    'node server.js',
+    { stdio: 'inherit', shell: true, cwd, env }
   )
   await waitFor(500)
-  return instance.pid
+  return server.pid
 }
 
 describe('Home', () => {
-  let pid
+  let chromedriver, serverPid, browser
   beforeAll(async () => {
-    pid = await startApp()
+    chromedriver = spawn('chromedriver', { stdio: 'inherit' })
+    serverPid = await startApp()
+    browser = wd.promiseRemote('http://localhost:9515/')
+    await browser.init({browserName: 'chrome'})
   })
   afterAll(async () => {
-    fkill(pid)
+    await browser.quit()
+    try {
+      fkill(serverPid)
+      fkill(chromedriver.pid)
```

Comment:
without this, the test process doesn't exit. it doesn't completely kill chromedriver which is why the next line is needed.